.github/workflows/npmjs-cd.yml

```
@@ -1,74 +1,80 @@
+      - name: Read package info
+        id: pkg
+        shell: bash
+        run: |
+          set -euo pipefail
+          NAME="$(node -p "require('./package.json').name || ''")"
+          VERSION="$(node -p "require('./package.json').version || ''")"
+          # Fail fast if either value is empty
+          [ -n "$NAME" ] && [ -n "$VERSION" ]
+          echo "name=$NAME" >> "$GITHUB_OUTPUT"
+          echo "version=$VERSION" >> "$GITHUB_OUTPUT"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)